### DISCUSSION TITLE
Will I get full credit if I passed all test cases?

### DISCUSSION THREAD

**Discussion by jsjclink** [ANLAB-KAIST/KENSv3#53] [Project Q&A 1. Application Layer (Socket)] [answered]
Hello, I'm wonder if I can get full credit if I passed all test cases.

Should I need to add \n for all requests? or is it okay if I passed all test cases?

**Comment by Pusnow** [comment, marked as answer]
> I'm wonder if I can get full credit if I passed all test cases.

Yes

> Should I need to add \n for all requests? or is it okay if I passed all test cases?

We will not check `\n` in the packets, but we recommend to do so.
